PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD004.xml

```diff
@@ -0,0 +1,147 @@
+                    Ethiopia" funded by Gerda Henkel Stiftung. </p>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1014MD"/>
+                        <idno>MD004</idno>
+                    </msIdentifier>
+                    <msContents>
+                          <msItem xml:id="ms_i1">
+                           <title type="complete" ref="LIT3034RepCh314"/>
+                            <textLang mainLang="gez"/>
+                              <incipit>በስመ: አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብእ፡ ልሳነ፡ ምቀኛ፡ ወተንኮለኛ፡ ናሁ፡ ለዚባዎስሙ፡ 
+                                  በዝ፡ ድብ፡ ያዕቆብ፡ እምልሳነ፡ ሰብእ፡ ወነገደ፡ ያዕቆብ፡ ምድረ፡ </incipit>
+                              <explicit><unclear>ባህረ፡ ልሳነ፡ ፍስሐ፡ ገፁ፡ ተማህፀንኩ፡ በጸሎትከ፡ ሞተ፡ አቡ፡ ወዕቀበኒ፡ ዘበልማድ፡ እምዝ፡ ቆንተንቶሙ፡ ብከ፡ እስከተሐዉሱ፡ እምባእድ፡ 
+                                  ጥቀ፡ ሠናያቲከ፡ ብዙህ፡ እመ፡ አዕፁከ፡ እዴሁ</unclear> </explicit>
+                        </msItem>
+                        </msContents>
+                    <physDesc>
+                        <objectDesc form="Leporello">
+                        <supportDesc>
+                            <support>
+                                <material key="parchment"/>
+                            </support>
+                            <extent>
+                                <measure unit="leaf" quantity="28">28</measure>
+                 
+                            </extent>
+                            <foliation> The manuscript is not foliated. The numbers used in this description refer the panels.
+                               </foliation>
+                            </supportDesc>
+                            </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>The handwriting is very difficult to read; the manuscript was copied by <persName
+                                    ref="PRS14417MarigetaDamseEndalew" role="scribe"> </persName>
+                                </desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication"> The word ተማህፀንኩ and personal names are rubricated.
```

Comment:
I always try to mark up Geez words like this: `<foreign xml:lang="gez"> ተማህፀንኩ </foreign>`